guides/assistant.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "AI Assistant"
+description: "Help users succeed with your product and find answers faster"
+icon: "bot"
+---
+
+<Info>
```

Comment:
I think we should try to remove sections that tell people to contact sales. I'm not sure always giving a CTA to contact sales lands well with developers